idea-plugin/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'java'
+apply plugin: 'com.palantir.external-publish-intellij'
+
+intellij {
+    pluginName = "gradle-consistent-versions"
+    updateSinceUntilBuild = true
+    version = "2024.1"
+    plugins = ['java', 'org.jetbrains.plugins.gradle']
+}
+
+patchPluginXml {
+    pluginDescription = "Language plugin for version.props allowing for auto-lock updating"
+    version = "1.0.0" // For some reason project.version is unspeificed and thus this fails
```

Comment:
Not sure why am needing to hard code this currently @CRogers any ideas?

Reply:
https://github.com/palantir/gradle-consistent-versions/pull/1235

Reply:
awesome - merged that in and it seems to be working!